packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v2.3.0-deploy-contract-from-json.test.ts

```diff
@@ -226,13 +231,9 @@ test(testCase, async (t: Test) => {
         },
         nonce: 2,
       });
-      t2.ifError(setNameOutInvalid.transactionReceipt);
+      fail("it should not reach here");
```

Comment:
The fail message should be like, "getContractInfoKeychain method should fail for new name, but recieved" + setNameOutInvalid.transactionReceipt (same change at other similar places)

Reply:
> The fail message should be like, "getContractInfoKeychain method should fail for new name, but recieved" + setNameOutInvalid.transactionReceipt (same change at other similar places)

@jagpreetsinghsasan I applied the fixes in all 3 places where it was applicable. Marking as resolved.